backend/typescript/utilities/logger.ts

```diff
@@ -0,0 +1,26 @@
+import * as winston from "winston";
+
+const logger: winston.Logger = winston.createLogger({
+  level: "info",
+  format: winston.format.combine(
+    winston.format.timestamp(),
+    winston.format.json(),
+  ),
+  transports: [
+    new winston.transports.File({ filename: "error.log", level: "error" }),
+    new winston.transports.File({ filename: "combined.log" }),
+  ],
+});
+
+if (process.env.NODE_ENV !== "production") {
+  logger.add(
+    new winston.transports.Console({
+      format: winston.format.combine(
+        winston.format.timestamp(),
+        winston.format.json(),
+      ),
+    }),
+  );
+}
```

Comment:
Log into the files `error.log` and `combined.log`, and also to the console when not in prod.

Format of log message:
```
{"message":"......","level":"error","timestamp":"2021-03-13T03:26:25.269Z"}
```